cmd/grpcurl/grpcurl.go

```diff
@@ -407,7 +422,14 @@ func main() {
 			opts = append(opts, grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(*maxMsgSz)))
 		}
 		var creds credentials.TransportCredentials
-		if !*plaintext {
+		if *plaintext {
+			if *authority != "" {
+				opts = append(opts, grpc.WithAuthority(*authority))
+			}
+		} else if *usealts {
+			creds = alts.NewClientCreds(alts.DefaultClientOptions())
```

Comment:
Would we not want to support some options here, like specifying the trusted/expected server service account or allowing a custom handshaker address?

Reply:
Good point, I don't use these arguments but will add in case someone needs to.

Reply:
Added in new commit